system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -199,6 +199,13 @@ component displayName="Preside Object Service" {
 
 		if ( !args.allowDraftVersions && !args.fromVersionTable && objectIsVersioned( args.objectName ) ) {
 			args.extraFilters.append( _getDraftExclusionFilter( args.objectname ) );
+
+			if ( ( arguments.selectManyToMany ?: false ) and !isEmpty( arguments.relationshipTable ?: "" ) and !$getRequestContext().isAdminRequest() ) {
```

Comment:
What is the purpose of only doing this for non-admin requests? We already have the `!args.allowDraftVersions` check in the containing `if` condition. Is this not sufficient?